openownership/data-standard
# Converted paths -> text elements in SVGs, fixes #429

# Overview

- What does this pull request do?

Edits 9 SVG files

- How can a reviewer test or examine your changes?

Build the docs, check that the SVGs look as expected in:

- schema/guidance/repr-beneficial-ownership.html
- schema/guidance/repr-state-owned-enterprises.html

And when you view the image in a browser, you should be able to select the text and copy it as text.

- Who is best placed to review it?

@rhiaro  

(Closes/Relates to) issue: # 429

## Translations

      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

n/a
